proto/viam/app/data/v1/data.proto

```diff
@@ -173,3 +185,54 @@ message DeleteBinaryDataByIDsRequest {
 message DeleteBinaryDataByIDsResponse {
   uint64 deleted_count = 1;
 }
+
+// Annotation specifies the ways in which we can associate annotations with data, e.g. with a text tag or image bounding box
+message Annotation {
+  string tag = 1;
+}
+
+// AddAnnotationsToBinaryDataByFileIDsRequest requests adding annotations to the files when file IDs are provided
+message AddAnnotationsToBinaryDataByFileIDsRequest {
+  repeated string file_ids = 1;
+  repeated Annotation annotations = 2;
+}
+
+// AddAnnotationsToBinaryDataByFileIDsResponse return IDs of added annotations in order to pass it to RemoveAnnotations
```

Comment:
supernit: perhaps "which can be passed to RemoveAnnotationsFromBinaryDataByFileIDs"

Reply:
done!